pkg/alertmanager/multitenant.go

```diff
@@ -86,9 +86,10 @@ const (
 var (
 	statusTemplate *template.Template
 
-	errInvalidExternalURL         = errors.New("the configured external URL is invalid: should not end with /")
-	errShardingLegacyStorage      = errors.New("deprecated -alertmanager.storage.* not supported with -alertmanager.sharding-enabled, use -alertmanager-storage.*")
-	errShardingUnsupportedStorage = errors.New("the configured alertmanager storage backend is not supported when sharding is enabled")
+	errInvalidExternalURL                  = errors.New("the configured external URL is invalid: should not end with /")
+	errShardingLegacyStorage               = errors.New("deprecated -alertmanager.storage.* not supported with -alertmanager.sharding-enabled, use -alertmanager-storage.*")
+	errShardingUnsupportedStorage          = errors.New("the configured alertmanager storage backend is not supported when sharding is enabled")
+	errZoneAwarenessEnabledWithoutZoneInfo = errors.New("the configured alertmanager has zone aware enabled but zone is not set")
```

Comment:
**nit:** We use the phrase zone awareness everywhere else, I think it would make sense to keep it consistent here as well

```suggestion
	errZoneAwarenessEnabledWithoutZoneInfo = errors.New("the configured alertmanager has zone awareness enabled but zone is not set")
```